src/reset-password/tests/ResetPasswordPage.test.jsx

```diff
@@ -157,6 +157,55 @@ describe('ResetPasswordPage', () => {
 
   // ******** miscellaneous tests ********
 
+  it('check password value on blur and enable show password field', () => {
+    const resetPasswordPage = mount(reduxWrapper(<IntlResetPasswordPage {...props} />));
+    resetPasswordPage.find('input#newPassword').simulate('change', { target: { value: 'aziz156@', name: 'newPassword' } });
+    resetPasswordPage.find('input#newPassword').simulate('blur', { target: { value: 'aziz156@', name: 'newPassword' } });
+    expect(resetPasswordPage.find('input#newPassword').props().value).toEqual('aziz156@');
+  });
+  it('should not call validation when typing and click on show icon button', () => {
+    const resetPasswordPage = mount(reduxWrapper(<IntlResetPasswordPage {...props} />));
+    resetPasswordPage.find('input#newPassword').simulate('change', { target: { value: 'aziz156@', name: 'newPassword' } });
+    resetPasswordPage.find('button[aria-label="Show password"]').at(0).simulate('click');
+    resetPasswordPage.find('button[aria-label="Hide password"]').at(0).simulate('blur');
+    expect(resetPasswordPage.find('input#newPassword').prop('type')).toEqual('text');
+  });
+  it('should call validation click on show icon button and then focus out', () => {
+    const resetPasswordPage = mount(reduxWrapper(<IntlResetPasswordPage {...props} />));
+    resetPasswordPage.find('input#newPassword').simulate('focus', { target: { value: 'aziz156@', name: 'newPassword' } });
+    resetPasswordPage.find('input#newPassword').simulate('blur', { relatedTarget: { name: 'password' } });
+    resetPasswordPage.find('button[aria-label="Show password"]').at(0).simulate('click');
+    expect(resetPasswordPage.find('input#newPassword').prop('type')).toEqual('text');
+  });
+  it('show spinner when api call is pending', () => {
+    store.dispatch = jest.fn(store.dispatch);
+    props = {
+      status:
+      'token-pending',
+      match: {
+        params: { token: '1c-bmjdkc-5e60e084cf8113048ca7' },
+      },
+    };
+    mount(reduxWrapper(<IntlResetPasswordPage {...props} />));
+    expect(store.dispatch).toHaveBeenCalledWith(validateToken(props.match.params.token));
+  });
+  it('should redirect the user to Reset password email screen ', async () => {
+    props = {
+      status:
+      'password-reset-error',
```

Comment:
Remove this hardcoded const and import this from the constant file.

Reply:
addressed